js/scenes/Debug.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+} from 'react-native'
+
+import Header from '../components/Header';
+import Compass from '../components/Compass';
+import { defaultTextStyle } from '../../config';
+
+export default class Debug extends Component {
+  constructor(props) {
+    super(props);
+
+    this.navigator = this.props.navigator;
+    this.onBackPress = this.onBackPress.bind(this);
+
+    this.state = {
+      latitude: 0,
+      longitude: 0
+    };
+  }
+
+  onBackPress() {
+    this.navigator.pop();
+  }
+
+  // Watch for position for testing the compass
+  componentWillUnmount() { 
+    navigator.geolocation.clearWatch(this.watchID); 
+  }
+
+  componentDidMount() { 
+    navigator.geolocation.getCurrentPosition( (position) => { 
```

Comment:
Yes, I think we will have an unique source of truth for the location, but until we have that, (we marked as one of the components), I was using that cheap way of having location for testing the component.